### ISSUE TITLE
Forecast Holidays

### ISSUE THREAD

**Issue by oguchihy** [business-science/modeltime#203]
Holidays need to be accounted for in the forecast; not just as a regressor in training.  For example, a forecast for Christmas Day 2022 should be zero if Christmas is an observed holiday

**Comment by LeoTimmermans** [comment]
> Holidays need to be accounted for in the forecast; not just as a regressor in training. For example, a forecast for Christmas Day 2022 should be zero if Christmas is an observed holiday

I disagree. I can think of several scenario's where a forecast would spike on days marked as holiday.
You can always do some post processing on your forecasts, if you need Christmas Day to be zero.

**Comment by spsanderson** [comment]
I disagree as well, Christmas say at a Chinese Restaurant in NY is going to be very busy.
